src/objects/Mesh.js

```diff
@@ -206,7 +206,7 @@ THREE.Mesh.prototype.raycast = ( function () {
 						distance: distance,
 						point: intersectionPoint,
 						face: new THREE.Face3( a, b, c, THREE.Triangle.normal( vA, vB, vC ) ),
-						faceIndex: null,
+						faceIndex: i/3,
```

Comment:
in case 
`if ( attributes.index !== undefined )`
`face.a / 3` is not that I need, because `face.a = index + indices[ i ];` => user have no valid method to determine, which triangle number got a hit.